src/blenderbim/blenderbim/bim/module/pset/calc_quantity_function_mapper.py

```diff
@@ -0,0 +1,10 @@
+mapper = {
```

Comment:
Can put this in a helper.py instead?

Reply:
I don't exactly get it, do you mean rename the file into "helper.py"? :)
I put the mapper dict in a separate file so it's easier to help

Reply:
ah ok, sure then :) never mind